tests/guardrails/test_guardrails_types.py

```
@@ -191,131 +212,78 @@ def test_prefers_the_most_specific_source(self, result, expected):
-    # The verdicts below come from rails IORails now runs, so these are live payloads rather
-    # than anticipated ones.
-
+    # ``_regex_outcome`` builds metadata from its whole ``RegexDetectionResult``, so the
+    # text being checked and the substrings that matched are both in there.
     REGEX_VERDICT = {
-        "allowed": False,
         "is_match": True,
         "text": "my card is 4111 1111 1111 1111",
         "detections": ["4111 1111 1111 1111"],
         "source": "input",
     }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)